src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -18,13 +23,20 @@ export const useTradeRoutes = (): {
   const buyAsset = getValues('buyAsset')
   const sellAsset = getValues('sellAsset')
   const assets = useSelector(selectAssets)
-  const feeAsset = assets['eip155:1/slip44:60']
+  const feeAsset = assets[ETHEREUM_CAIP19]
 
   const setDefaultAssets = useCallback(async () => {
     try {
       const [sellAssetId, buyAssetId] = getDefaultPair()
       const sellAsset = assets[sellAssetId]
-      const buyAsset = assets[buyAssetId]
+      // TODO: Actually we only support ERC20 pairs, but for example we will need to support cosmos pairs soon
+      const buyAsset =
+        defaultBuyAssetId &&
+        assets[defaultBuyAssetId]?.chain === ChainTypes.Ethereum &&
+        assets[defaultBuyAssetId]?.caip19 !== ETHEREUM_CAIP19
+          ? assets[defaultBuyAssetId]
+          : assets[buyAssetId]
+
```

Comment:
The ETH asset page wasn't specified in the specs, but we should never see the ETH => ETH trade pair

Also, there are some chain specific code @cjthompson as you seen, because currently we are only supporting ETH trade pairs, when cosmos will be merged and working, we should work on a way to whitelist multiple chains there!